framework/src/problems/FEProblemBase.C

```diff
@@ -4245,6 +4245,77 @@ FEProblemBase::execMultiAppTransfers(ExecFlagType type, Transfer::DIRECTION dire
+
+    // output transfer information
+    unsigned int max_name_length = 4;
+    unsigned int max_type_length = 4;
+    unsigned int max_multiapp_name_length = 13;
+    for (const auto & transfer : transfers)
+    {
+      if (transfer->name().length() > max_name_length)
+        max_name_length = transfer->name().length();
+      if (transfer->type().length() > max_type_length)
+        max_type_length = transfer->type().length();
+      auto multiapp_transfer = dynamic_cast<MultiAppTransfer *>(transfer.get());
+      if (multiapp_transfer)
+      {
+        if (direction == MultiAppTransfer::TO_MULTIAPP)
+        {
+          unsigned int l = multiapp_transfer->getToMultiApp()->name().length();
+          if (l > max_multiapp_name_length)
+            max_multiapp_name_length = l;
+        }
+        else if (direction == MultiAppTransfer::FROM_MULTIAPP)
+        {
+          unsigned int l = multiapp_transfer->getFromMultiApp()->name().length();
+          if (l > max_multiapp_name_length)
+            max_multiapp_name_length = l;
+        }
+        else
+        {
+          unsigned int l1 = multiapp_transfer->getFromMultiApp()->name().length();
+          unsigned int l2 = multiapp_transfer->getToMultiApp()->name().length();
+          if (l1 + l2 + 2 > max_multiapp_name_length)
+            max_multiapp_name_length = l1 + l2 + 2;
+        }
+      }
+    }
+    const unsigned int space = 2;
+    _console << std::left << std::setw(max_name_length + space) << "Name";
+    _console << std::left << std::setw(max_type_length + space) << "Type";
+    _console << std::left << std::setw(max_multiapp_name_length) << "MultiApp name\n";
+    const unsigned int total_length =
+        max_name_length + max_type_length + max_multiapp_name_length + space * 2;
+    for (unsigned int i = 0; i < total_length; ++i)
+      _console << '-';
+    _console << '\n';
+
+    for (const auto & transfer : transfers)
+    {
+      _console << std::left << std::setw(max_name_length + space) << transfer->name();
+      _console << std::left << std::setw(max_type_length + space) << transfer->type();
+      auto multiapp_transfer = dynamic_cast<MultiAppTransfer *>(transfer.get());
+      if (multiapp_transfer)
+      {
+        std::string name;
+        if (direction == MultiAppTransfer::TO_MULTIAPP)
+          name = multiapp_transfer->getToMultiApp()->name();
+        else if (direction == MultiAppTransfer::FROM_MULTIAPP)
+          name = multiapp_transfer->getFromMultiApp()->name();
+        else
+        {
+          name += multiapp_transfer->getFromMultiApp()->name();
+          name += "=>";
+          name += multiapp_transfer->getToMultiApp()->name();
+        }
+        _console << std::left << std::setw(max_multiapp_name_length) << name;
+      }
+      _console << "\n";
+    }
+    for (unsigned int i = 0; i < total_length; ++i)
+      _console << '-';
+    _console << std::endl;
+
```

Comment:
I don't think any of this transfer information should come out by default.  Let's put it behind a `TIMED_SECTION`.  We really only care if it's taking too long.

Reply:
This is similar as the backup/restore message. This summarizes the transfer information. If we do not have this, there could be errors in the input that are really hard to detect. I will add `TIMED_SECTION` for individual transfers, but I think we should keep this. I know the codes here are a bit long.